lib/cachex/actions/put.ex

```diff
@@ -32,7 +32,7 @@ defmodule Cachex.Actions.Put do
     record = entry_now(key: key, ttl: expiry, value: value)
 
     Locksmith.write(cache, [key], fn ->
-      Actions.write(cache, record)
+      Actions.write(cache, [record])
```

Comment:
This looks strange, but underneath `:ets.insert/2` will accept both a list or a single value

https://www.erlang.org/doc/man/ets#insert-2

Between making the `@spec` extra long or just sending the data through as a list I opted for the smaller change of wrapping the singular value in a list.

Reply:
I'm assuming there's a performance impact of this? I know it seems minor, but having it in a hot path could be relevant.

I'm open to making dialyzer happy, but in the case it makes things "worse" I'm not a fan.